apps/api/src/app/step-schemas/e2e/get-step-schema.e2e.ts

```diff
@@ -0,0 +1,283 @@
+      expect(data.controls.properties.subject.type).to.equal('string');
+      expect(data.controls.properties).to.have.property('body');
+      expect(data.controls.properties.body.type).to.equal('string');
+      expect(data.controls.properties).to.have.property('avatar');
+      expect(data.controls.properties.avatar.type).to.equal('string');
+      expect(data.controls.properties.avatar.format).to.equal('uri');
+      expect(data.controls.properties).to.have.property('primaryAction');
+      expect(data.controls.properties.primaryAction.type).to.equal('object');
+      expect(data.controls.properties.primaryAction.properties).to.have.property('label');
+      expect(data.controls.properties.primaryAction.properties.label.type).to.equal('string');
+      expect(data.controls.properties.primaryAction.properties).to.have.property('redirect');
+      expect(data.controls.properties.primaryAction.properties.redirect.type).to.equal('object');
+      expect(data.controls.properties.primaryAction.required).to.deep.equal(['label']);
+      expect(data.controls.properties).to.have.property('secondaryAction');
+      expect(data.controls.properties.secondaryAction.type).to.equal('object');
+      expect(data.controls.properties.secondaryAction.properties).to.have.property('label');
+      expect(data.controls.properties.secondaryAction.properties.label.type).to.equal('string');
+      expect(data.controls.properties.secondaryAction.properties).to.have.property('redirect');
+      expect(data.controls.properties.secondaryAction.properties.redirect.type).to.equal('object');
+      expect(data.controls.properties.secondaryAction.required).to.deep.equal(['label']);
+      expect(data.controls.properties).to.have.property('data');
+      expect(data.controls.properties.data.type).to.equal('object');
+      expect(data.controls.properties.data.additionalProperties).to.be.true;
+      expect(data.controls.properties).to.have.property('redirect');
+      expect(data.controls.properties.redirect.type).to.equal('object');
+      expect(data.controls.properties.redirect.properties).to.have.property('url');
+      expect(data.controls.properties.redirect.properties.url.type).to.equal('string');
+      expect(data.controls.properties.redirect.properties).to.have.property('target');
+      expect(data.controls.properties.redirect.properties.target.type).to.equal('string');
+      expect(data.controls.required).to.deep.equal(['body']);
+      expect(data.controls.additionalProperties).to.be.false;
+      expect(data.controls.description).to.equal(
+        'Output of the step, including any controls defined in the Bridge App'
+      );
+
+      expect(data.variables.type).to.equal('object');
+      expect(data.variables.description).to.equal(
+        // eslint-disable-next-line max-len
+        'Variables that can be used with Liquid JS Template syntax. Includes subscriber attributes, payload variables, and supports liquid filters for formatting.'
```

Comment:
This seems quite specific, and while it may not break any functionality, it could lead to potential issues down the line.